azerbaijani/nağıl

```diff
@@ -0,0 +1,7 @@
+Biri var idi, biri yox idi. Bir ayı, günü keçmiş kişi var idi. Bir arvadı var idi. Bunların yeddi oğlanları var idi. Amma heç qız uşaqları yox idi. Yeddi qardaşlar bir bacıları olmadığından çox qəmgin idilər. Bir gün yeddi qardaşlar analarına dedilər:
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.